app/controllers/focus_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class FocusController < ApplicationController
```

Comment:
Controller names should be pluralized. Don't forget to update the file name too!!
```suggestion
class FocusesController < ApplicationController
```